content/automate/ManageIQ/Transformation/Common.class/__methods__/assesstransformationcleanup.rb

```diff
@@ -0,0 +1,32 @@
+module ManageIQ
+  module Automate
+    module Transformation
+      module Common
+        class AssessTransformationCleanup
+          def initialize(handle = $evm)
+            @handle = handle
+          end
+
+          def main
+            task = @handle.vmdb(:service_template_transformation_plan_task).find_by(:id => @handle.root['service_template_transformation_plan_task_id'])
+            raise 'No task found. Exiting' if task.nil?
+            @handle.log(:info, "Task: #{task.inspect}") if @debug
```

Comment:
`@debug` is referenced here but never defined.

Reply:
Fixed with default value to `false`.